crypto/ppccap.c

```diff
@@ -68,11 +68,12 @@ int bn_mul_mont(BN_ULONG *rp, const BN_ULONG *ap, const BN_ULONG *bp,
      * no opportunity to figure it out...
      */
 
-    if (num == 6)
+    if (num == 6) {
```

Comment:
Is it really that stuupid?

Reply:
Just a hint saying the else could be misinterpreted.